ifStatement.js

```diff
@@ -33,6 +39,17 @@ Steps:
     - If either of the conditions is false, print the message Hello ${userName}, I'm sorry but it seems you're not authorized to access the restricted area.
 ***********************************************************/
 // TODO: ADD YOUR CODE BELOW
+let YOUR_NAME = "admin";
```

Comment:
`YOUR_NAME` is not a variable name. It's written in the description to instruct you to add your name as a value, like this:
username = "Mustafa"